lib/flat_map/errors.rb

```diff
@@ -49,7 +49,7 @@ def empty?
     end
 
     # Overridden to add suffixing support for mappings of mappers with name suffix
-    def add(attr, *args)
+    def add(attr, type, **options)
```

Comment:
Confirmed that [`ActiveModel::Errors#add`](https://github.com/rails/rails/blob/v6.1.7.6/activemodel/lib/active_model/errors.rb#L404) in Rails 6 shares this same type signature. Nice find.